docs/admin/reference/ssh_access.rst

```diff
@@ -0,0 +1,190 @@
+Logging in via SSH
+==================
+
+.. _server SSH access:
+
+.. note::
+  You can access your *Application Server* and *Monitor Server* via SSH by
+  using either the `ssh app` or `ssh mon` commands (respectively) from an
```

Comment:
Nit: `ssh app` / `ssh mon` (currently rendered as italics due to single tic)

Structurally, I would put the instructions after the caveat ("Generally, you should avoid") and do it as regular text rather than in note format, for two reasons:
- as a general principle, I think it makes sense to show warnings/caveats before specific instructions (i.e. I would prefer "Warning: Pushing the button makes things explode. Step 1: Push the button" over "Step 1: Push the button. Warning: Pushing the button makes things explode" :)
- in this instance, we also have two "tip" call-outs competing for visual attention, so there's a risk of things getting a bit cluttered.

Reply:
Great catch! And I agree, that explanation makes sense, and after making the change I prefer it myself.